### ISSUE TITLE
Screen reader does not announce number of modules mentioned next to packages in Intellisense view

### ISSUE THREAD

**Issue by eclectir** [Microsoft/PTVS#2237]
Screen reader announces names of the packages, but not the number of modules in parenthesis.

**Comment by huguesv** [comment]
This has been fixed.
